mainWidget/main_widget.yaml

```diff
@@ -203,8 +202,8 @@ slots:
                             updateOnWindowResize: true
                             watchOverflow: true
                           style:
-                            padding-left: 50px
-                            padding-right: 50px
+                            #padding-left: 50px
+                            #padding-right: 50px
```

Comment:
Please remove those lines completely, not just add a hashtag

Reply:
removed